screens/UserListScreen.tsx

```diff
@@ -49,6 +66,10 @@ export const UserListScreen = () => {
     );
```

Comment:
[non-blocking]
今回のスコープではないですが、`userData === []`は常にfalseになるようです。
```
This condition will always return 'false' since JavaScript compares objects by reference, not value.ts(2839)
```
```.sh
> const emptyArray = []
undefined
> [] === []
false
```

Reply:
arrayの空判定はlength使うと良いです
```.sh
> [].length
0
```

Reply:
常に false になるんですね💦知りませんでした。
ありがとうございます🙇‍♀️